<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCA</class>
 <widget class="QMainWindow" name="PCA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(146, 208, 80);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Return">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>189</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 87, 35);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Return to
Main Menu</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>691</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="pushButton_ShowOutput">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>189</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 87, 35);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Show
Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SaveTestResult">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>189</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 87, 35);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Save
Test Result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>987</x>
      <y>-3</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Minimize">
    <property name="geometry">
     <rect>
      <x>946</x>
      <y>-3</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
